### PR TITLE
Fix valset loading for videos

### PR DESCRIPTION
**Pull request by zhuohaoyu**
Currently the loading of validation set of video data is bugged, because video_key and video_fps are not actually passed to RLHFDataset in the dataloader.

### CODE REVIEW

**Review by hiyouga — APPROVED** [review]
LGTM
